docs/conf/portland/2019/news/cfp-reminder.rst

```diff
@@ -0,0 +1,44 @@
+:template: {{year}}/generic.html
+
+.. post:: January 7, 2019
+   :tags: portland-2019, website, cfp, tickets
+
+
+CFP Reminder & Tickets on Sale
+==============================
+
+Happy New Year, fellow documentarians!
+
+As we get ready to start the year, we're happy to share the latest update about the Portlant conference. We are extending our `call for proposals <http://www.writethedocs.org/conf/portland/2019/cfp/>`_ until **January 31st**, and `tickets for the conference are officially on sale <http://www.writethedocs.org/conf/portland/2019/tickets/>`_. Read on for more details on both!
+
+Submit your talk by January 31st
+--------------------------------
+
+As we are aware that many documentarians were away for the holidays, we'd like to give everyone an opportunity to submit their talks.
+The Call for Proposals will remain open until **Thursday, January 31st at midnight PST**. Check out the `CFP page <http://www.writethedocs.org/conf/portland/2019/cfp/#submit-your-proposal>`_ for more details about what we are looking for and how to submit your proposal.
+
+And if you've already got an idea, but you’re having a hard time getting your proposal to come together, you can always ask for feedback on the `WTD Slack <http://www.writethedocs.org/slack/>`_. The community is there to help!
```

Comment:
I like this actually, since it introduces people to Slack if they don't have it (assuming that people who are already on our Slack won't click the link)

Reply:
Not sure I agree with the assumption, but sure. :-)

Reply:
If we link directly to the slack, people who aren't members can't sign up, which is a shit experience.